www-cronrobot/app/views/dashboard/schedulers/index.html.erb

```diff
@@ -1,63 +1,56 @@
-scheds!
+<% if @schedulers.empty? %>
+<h1>Create your first Scheduler</h1>
+<% else %>
+<h1>Schedulers</h1>
+<% end %>
 
-<ul>
-    <li>
-        <a href="/dashboard/schedulers/new/SchedulerSocketPing">
-            Create Socket ping scheduler
-        </a>
-    </li>
-    <li>
-        <a href="/dashboard/schedulers/new/SchedulerHttp">
-            Create HTTP scheduler
-        </a>
-    </li>
-    <li>
-        <a href="/dashboard/schedulers/new/SchedulerSsh">
-            Create SSH scheduler
-        </a>
-    </li>
-</ul>
+<form id="scheduler-select">
+    <select required="true" class="select-input">
```

Comment:
i don't think it's great in terms of UI/UX to have it always displayed. It's always taking space even when you don't want to select a scheduler to create

would be better if the select just appears when you click create, then you select the type perhaps.